src/main/resources/edu/cornell/kfs/coa/businessobject/datadictionary/ObjectCode.xml

```diff
@@ -25,304 +25,254 @@
-              <bean parent="FieldDefinition" p:attributeName="financialBudgetAggregationCd"/>
-              <bean parent="FieldDefinition" p:attributeName="finObjMandatoryTrnfrelimCd"/>
-              <bean parent="FieldDefinition" p:attributeName="financialFederalFundedCode"/>
-              <bean parent="FieldDefinition" p:attributeName="nextYearFinancialObjectCode"/>
+                <bean parent="FieldDefinition" p:attributeName="universityFiscalYear"/>
+                <bean parent="FieldDefinition" p:attributeName="chartOfAccountsCode"/>
+                <bean parent="FieldDefinition" p:attributeName="financialObjectCode"/>
+                <bean parent="FieldDefinition" p:attributeName="financialObjectCodeName"/>
+                <bean parent="FieldDefinition" p:attributeName="financialObjectCodeShortName"/>
+                <bean parent="FieldDefinition" p:attributeName="reportsToChartOfAccountsCode"/>
+                <bean parent="FieldDefinition" p:attributeName="reportsToFinancialObjectCode"/>
+                <bean parent="FieldDefinition" p:attributeName="financialObjectTypeCode"/>
+                <bean parent="FieldDefinition" p:attributeName="financialObjectLevelCode"/>
+                <bean parent="FieldDefinition" p:attributeName="extension.cgReportingCode"/>
+                <bean parent="FieldDefinition" p:attributeName="financialObjectSubTypeCode"/>
+                <bean parent="FieldDefinition" p:attributeName="extension.sunyObjectCode"/>
+                <bean parent="FieldDefinition" p:attributeName="extension.financialObjectCodeDescr"/> 
+                <bean parent="FieldDefinition" p:attributeName="historicalFinancialObjectCode"/>
+                <bean parent="FieldDefinition" p:attributeName="active"/>
+                <bean parent="FieldDefinition" p:attributeName="financialBudgetAggregationCd"/>
+                <bean parent="FieldDefinition" p:attributeName="finObjMandatoryTrnfrelimCd"/>
+                <bean parent="FieldDefinition" p:attributeName="financialFederalFundedCode"/>
+                <bean parent="FieldDefinition" p:attributeName="nextYearFinancialObjectCode"/>
             </list>
-          </property>
-        </bean>
-      </list>
-    </property>
-  </bean>
+        </property>
+    </bean>
 
 <!-- Business Object Lookup Definition -->
 
-
-  <bean id="ObjectCode-lookupDefinition" parent="ObjectCode-lookupDefinition-parentBean"/>
-
-  <bean id="ObjectCode-lookupDefinition-parentBean" abstract="true" parent="LookupDefinition">
-    <property name="title" value="Object Code Lookup"/>
-    
-    <property name="defaultSort">
-      <bean parent="SortDefinition">
-        <property name="attributeNames">
-          <list>
-            <value>financialObjectCode</value>
-          </list>
+    <bean id="ObjectCode-lookupDefinition" parent="ObjectCode-lookupDefinition-parentBean"/>
+    <bean id="ObjectCode-lookupDefinition-parentBean" abstract="true" parent="LookupDefinition"

```

Comment:
KualiCo has a third bean in this lookupDefinition hierarchy, which might have been needed for Access Security or some other related override. Does that third bean need to be added here now, or are you planning to address that in the follow-up PR?

Reply:
Also, one of these base beans has a multipleValuesResultSetLimit of -1 (unlimited?). Should that be added in here yet?

Reply:
Good catches.  I have added both items noted because the multipleValuesResultSetLimit of -1 is set in a base parent bean that would not be present should the third bean you noted as being missing from this lookupDefinition hierarchy not be present.